packages/runtime/container-runtime/src/summary/summarizerNode/summarizerNode.ts

```diff
@@ -386,9 +380,6 @@ export class SummarizerNode implements IRootSummarizerNode {
 	public async refreshLatestSummary(
 		proposalHandle: string | undefined,
 		summaryRefSeq: number,
-		fetchLatestSnapshot: () => Promise<IFetchSnapshotResult>,
```

Comment:
Please update comment above that says:

> Otherwise, we fetch the latest snapshot and update latest summary based off of that.

Reply:
Looks like `proposalHandle` will always be defined now since you deleted the call for blindly refreshing from the server.

Reply:
Also case 3 in the comment is incorrect now

Reply:
[AB#5420](https://dev.azure.com/fluidframework/internal/_workitems/edit/5420)

Reply:
Updated the comments